lib/app.js

```diff
@@ -66,6 +66,18 @@ app.use(require('fh-wfm-user/lib/cloud/middleware/validateSession')(mediator, mb
 var syncOptions =   config.get("syncOptions");
 syncOptions.dataCollisionHandler = globalCollisionHandler;
 
+/**
+ *
+ * We are using mongoose models that have a `updatedAt` field.
+ *
+ * This can be used to detect if the record has changed.
+ *
+ * @param record
+ */
+syncOptions.hashFunction = function generateHash(record) {
```

Comment:
Going to move this only for mongoose as this seems to be breaking simple store implementation